packages/forms-web-app/tests/unit/controllers/guidance-pages.test.js

```diff
@@ -2,7 +2,7 @@ const guidancePagesController = require('../../../src/controllers/guidance-pages
 const { mockReq, mockRes } = require('../mocks');
 const { VIEW } = require('../../../src/lib/views');
 
-describe('controllers/appellant-submission/appeal-statement', () => {
+describe('controllers/appeal-householder-decision/appeal-statement', () => {
```

Comment:
The path for this hasn't change has it?

Reply:
No I'm just checking this out now, I think it's because of the way the tests are structured so they're using VIEWS.whatever to generate the url